lona/html/abstract_node.py

```diff
@@ -45,6 +45,16 @@ def get_all_node_classes():
             *AbstractNode._subclasses,
         ]
 
+    # comparisons #############################################################
+    def __eq__(self, other):
```

Comment:
Is this really necessary?

Reply:
What do you mean, the particular method or the entire feature? The feature gets used by the rendering test, to compare the server side node tree with the HTML that is shown in the browser.

Reply:
The question was about entire feature. Thanks for clarification.